addons/l10n_ro/account_tax_template.xml

```diff
@@ -1,9 +1,494 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
 <data noupdate="0">
+    
+    <record id="tvati" model="account.tax.template">
+        <field name="chart_template_id" ref="ro_chart_template"/>
+        <field name="name">TVA Taxare Inversa Livrări</field>
+        <field name="description">Ti-livr</field>
+        <field name="amount">0.000000</field>
+        <field name="type">percent</field>
+        <field name="type_tax_use">sale</field>
+        <field name="base_code_id" ref="vat_code_tvac_bazaTI"/>
+        <field name="base_sign">1.00</field>
+        <field name="ref_base_code_id" ref="vat_code_tvac_bazaTI"/>
+        <field name="ref_base_sign">-1.00</field>
+    </record>
+    
+    <record id="tvatip00" model="account.tax.template">
+        <field name="chart_template_id" ref="ro_chart_template"/>
+        <field name="name">TVA Taxare Inversa Achizitii 0%</field>
+        <field name="description">Ti-ach-0</field>
+        <field name="amount">1.000000</field>
+        <field name="type">percent</field>
+        <field name="type_tax_use">purchase</field>    
+        <field name="child_depend">True</field>
```

Comment:
While this will work, keep in mind that `<field name="child_depend">False</field>` will not give you the expected result (evaluated as True), the better notation would be `<field name="child_depend" eval="..." />`